splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/SpliceDataDictionary.java

```diff
@@ -367,6 +370,51 @@ public void createKeyColumnUseViewInSysIBM(TransactionController tc) throws Stan
         SpliceLogUtils.info(LOG, "View SYSKEYCOLUSE in SYSIBM is created!");
     }
 
+    private TabInfoImpl getNaturalNumbersTable() throws StandardException{
+        if(naturalNumbersTable==null){
+            naturalNumbersTable=new TabInfoImpl(new SYSNATURALNUMBERSRowFactory(uuidFactory,exFactory,dvf, this));
+        }
+        initSystemIndexVariables(naturalNumbersTable);
+        return naturalNumbersTable;
+    }
+
+    /**
+     * Populate SYSNATURALNUMBERS table with 1-2048.
+     *
+     * @throws StandardException Standard Derby error policy
+     */
+    private void populateSYSNATURALNUMBERS(TransactionController tc) throws StandardException{
+        TabInfoImpl ti = getNonCoreTI(SYSNATURALNUMBERS_CATALOG_NUM);
+        for (int i = 1; i <= 2048; i++) {
```

Comment:
why 2048? should be a constant in my opinion

Reply:
No particular reason for this, just that SQL Server has this table (similar purpose but not exactly the same ofc) with 2048 values.